charts/nri-bundle/README.md

```diff
@@ -21,6 +21,57 @@ This chart bundles multiple New Relic products helm-charts.
 | `kubeEvents.enabled`             | Install the [`nri-kube-events` chart][6] | false |
 | `logging.enabled`                | Install the [`newrelic-logging` chart][7] | false |
 
+## Configure Dependencies
+
+It is possible to configure dependencies values from this chart.  
+For example you can configure verboseLog variable of the newrelic-infrastructure chart installed ad a dependency as follows.
+
+``` yaml
+newrelic-infrastructure:
+  verboseLog: false
+```
+
+```
+$ helm upgrade --reuse-values -f values.yaml [RELEASE] [CHART]
+```
+
+The same approach can be followed to update any of the dependencies. [Official documentation](https://helm.sh/docs/chart_template_guide/subcharts_and_globals)
+
+### Monitor on host integrations
+
+If you wish to monitor services running on Kubernetes you can provide integrations
+configuration under integrations_config that it will passed down to the newrelic-infrastructure chart. 
+
+You just need to create a new entry where the "name" is the filename of the configuration file and the data is the content of
+the integration configuration. The name must end in ".yaml" as this will be the
+filename generated and the Infrastructure agent only looks for YAML files. 
+
+The data part is the actual integration configuration as described in the spec here:
+https://docs.newrelic.com/docs/integrations/integrations-sdk/file-specifications/integration-configuration-file-specifications-agent-v180
+
+In the following example you can see how to monitor a Redis integration with autodiscovery
+
+```yaml
+newrelic-infrastructure:
+  integrations_config:
+    - name: nri-redis.yaml
+      data:
+        discovery:
+          command:
+            # Run NRI Discovery for Kubernetes
+            # https://github.com/newrelic/nri-discovery-kubernetes
+            exec: /var/db/newrelic-infra/nri-discovery-kubernetes
+            match:
+              label.app: redis
+        integrations:
+          - name: nri-redis
+            env:
+              # using the discovered IP as the hostname address
+              HOSTNAME: ${discovery.ip}
+              PORT: 6379
```

Comment:
shouldn't we use ${discovery.port} here?

Reply:
I took the example from here:
 - https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-apps-services/tutorial-monitor-redis-running-kubernetes
That is the same reported here:
 - https://github.com/newrelic/helm-charts/blob/fcd1f52c1e931997bec3ca4e5874760d9c33f0e5/charts/newrelic-infrastructure/values.yaml#L137

I believe that since we are running in k8s and there are services in front of the endpoints same ports can be reused and therefore there is no need to change then. They common case is to have standard ports

Reply:
ok, makes sense 👍